src/session-pool.ts

```diff
@@ -919,9 +963,11 @@ export class SessionPool extends EventEmitter implements SessionPoolInterface {
     // Only create a new session if there are more waiters than sessions already
     // being created. The current requester will be waiter number _numWaiters+1.
     if (!this.isFull && this.totalPending <= this.totalWaiters) {
+      span.addEvent('pool is not full and more waiters than sessions');
```

Comment:
Remove this, "Creating session" should be enough

Reply:
Done, thanks!